components/src/preact/mutations/getMutationsTableData.ts

```diff
@@ -1,17 +1,52 @@
 import { type SubstitutionOrDeletionEntry } from '../../types';
 import { type ProportionInterval } from '../components/proportion-selector';
 
-export function getMutationsTableData(data: SubstitutionOrDeletionEntry[], proportionInterval: ProportionInterval) {
+export function getMutationsTableData(
+    data: SubstitutionOrDeletionEntry[],
+    baselineSubstitutionsOrDeletions: SubstitutionOrDeletionEntry[] | undefined,
+    overallVariantCount: number,
+    proportionInterval: ProportionInterval,
+) {
     const byProportion = (mutationEntry: SubstitutionOrDeletionEntry) => {
         return mutationEntry.proportion >= proportionInterval.min && mutationEntry.proportion <= proportionInterval.max;
     };
 
-    return data.filter(byProportion).map((mutationEntry) => {
+    const tableData = data.filter(byProportion).map((mutationEntry) => {
         return {
             mutation: mutationEntry.mutation,
             type: mutationEntry.type,
             count: mutationEntry.count,
             proportion: mutationEntry.proportion,
         };
     });
+
+    if (baselineSubstitutionsOrDeletions === undefined) {
+        return tableData;
+    }
+
+    const baselineMutationCounts = baselineSubstitutionsOrDeletions.reduce((baselineMutationCounts, mutation) => {
+        baselineMutationCounts.set(mutation.mutation.toString(), mutation.count);
```

Comment:
```suggestion
        baselineMutationCounts.set(mutation.mutation.code, mutation.count);
```
The code should be our identifier